HEC.FDA.Model/paireddata/GraphicalUncertainPairedData.cs

```diff
@@ -200,12 +202,26 @@ public IPairedData SamplePairedData(double probability, bool computeIsDeterminis
                 y[i] = _StageOrLogFlowDistributions[i].InverseCDF(probability);
             }
             PairedData pairedData = new PairedData(_NonExceedanceProbabilities, y, _metaData);
-            if (!pairedData.IsValidPerMetadata)
+            bool isMonotonicallyIncreasing = IsMonotonicallyIncreasing(pairedData);
+            if (!isMonotonicallyIncreasing)
             {
                 pairedData.ForceMonotonic();
             }
             return pairedData;
         }
+        //This method duplicates the IsArrayValid method functionality
+        //I don't like IsArrayValid, too complicated. 
+        private bool IsMonotonicallyIncreasing(IPairedData pairedData)
```

Comment:
I removed that method and the property rule using that method - the property rule only checked for monotonicity of the non-exceedance probabilities, which is not helpful, because we define those probabilities internally. We don't want to check for monotonicity of the stage or flow distributions because we know a priori they are pretty much guaranteed to be be non-monotonically increasing.